enterprise/admin.py

```diff
@@ -36,13 +37,54 @@ class EnterpriseCustomerBrandingConfigurationInline(admin.StackedInline):
     can_delete = False
 
 
+class ModelChoiceIntegerField(forms.ModelChoiceField):
+    """
+    A custom integer field for admin form, this field treats an integer field as a foreign key.
```

Comment:
Please use max column length to 79 for readability in doctstrings.
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html